pigz.c

```diff
@@ -3124,8 +3124,12 @@ local void show_info(int method, unsigned long check, length_t len, int cont) {
     if (g.stamp && !cont) {
         strcpy(mod, ctime(&g.stamp));
         now = time(NULL);
-        if (strcmp(mod + 20, ctime(&now) + 20) != 0)
-            strcpy(mod + 11, mod + 19);
+        if (strcmp(mod + 20, ctime(&now) + 20) != 0) {
+            // use memmove() for overlapping blocks rather than strcpy()
```

Comment:
That's a C++ comment. Please use a /* regular C comment */ .

Reply:
This is an auto-generated comment I suppose?

The suggestion does not make any sense to me.  All other comments in the `show_info()` function use the C++ style.